### DISCUSSION TITLE
Ordering: newest to oldest

### DISCUSSION THREAD

**Discussion by KlaymenDK** [olivierlacan/keep-a-changelog#503] [Ideas]
IMHO, the most recent changes are usually the most interesting. As such, they should be easily accessible.

Therefore, I propose that new entries (releases, if you will) should be added to the TOP of the list, rather than appended to the bottom. At the very least, the order should not be mandated by this spec, so that projects may decide for themselves which timeline direction they would prefer.

As long as the rest of the formatting is is per convention, parsers should have no trouble with this format, although perhaps with an additional statement for sorting. ...but us carbon-based units can avoid a little wear on our scroll wheels.

**Comment by olivierlacan** [comment]
This has been an implicit and strongly observed convention for as long as I can remember and we even have a mention of "Explanation of the recommended reverse chronological release ordering" in the example changelog but for some reason I can't find that mentioned anywhere else. 

I'm a bit confused that you seem to be proposing something then immediately saying the "spec" (this is not a spec, we provide guidelines) should not mandate order to let projects decide for themselves. 

Once again though, parsers are not the focus of this project. Any good convention will be friendly to parsers as a side-effect of being friendly to humans, but the reverse is not necessarily true.

**Comment by plocket** [comment]
It looks like this was added. I see it in "The latest version comes first." in the ["How do I make a good changelog" section](https://keepachangelog.com/en/1.0.0/#how). Might want to rephrase to "The most recent version comes first" to make the language a bit more plain and straightforward, but this issue might be take care of.

**Comment by olivierlacan** [comment]
This phrasing has been there for years.

I don't believe the "most recent" version is always the latest (in descending version order) so that's not quite what I meant. 

It's really up to maintainers to decide if the absolute latest *version* should always show up at the top. I know it's a frequent issue when discussing patch version releases from earlier branches (major or minor) which aren't the latest *versions* available but they are the latest *releases* available by date.
